src/schemas/ev42/ev42_rw.cxx

```diff
@@ -93,13 +93,6 @@ HDFWriterModule::init_hdf(hid_t hdf_file, std::string hdf_parent_name,
     return HDFWriterModule::InitResult::ERROR_INCOMPLETE_CONFIGURATION();
   }
   auto sourcename = str.v;
```

Comment:
`sourcename` is now unused, can be removed.

Reply:
:+1: